spring/spring4/spring4-remote/src/test/java/org/infinispan/spring/provider/sample/SampleHotrodServerLifecycleBean.java

```diff
@@ -46,8 +47,8 @@ public void afterPropertiesSet() throws Exception {
       cacheManager.defineConfiguration(remoteBackupCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       cacheManager.defineConfiguration(customCacheName, HotRodTestingUtil.hotRodCacheConfiguration().build());
       HotRodServerConfigurationBuilder hcb = new HotRodServerConfigurationBuilder();
-      hcb.port(15233);
-      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, hcb);
+      hcb.port(PORT);
+      hotrodServer = HotRodClientTestingUtil.startHotRodServer(cacheManager, PORT, hcb);
```

Comment:
@slaskawi I suppose we'll never have enough Spring tests to make it worth running in parallel, so a fixed port should be fine. But instead of hard-coding the port here, I would inject the properties file and parse the port from the `infinispan.client.hotrod.server_list` property.